docs/DeveloperGuide.md

```diff
@@ -684,3 +746,60 @@ testers are expected to do more *exploratory* testing.
    1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
 
 1. _{ more test cases …​ }_
+
+### Data archiving
+
+1. Archiving an application while all unarchived application shown
+    1. Prerequisites: List all applications using the `list` command. Multiple applications in the list.
+
+    2. Test case: `archive 1`<br>
+       Expected: First application is archived from the list. Details of the archived application shown in the status message.
+
+    3. Test case: `archive 0`<br>
+       Expected: No application is archived. Error details shown in the status message. Status bar remains the same.
```

Comment:
what is status bar tho?

Reply:
Not sure tho, inherited from the delete manual testing. Since we didn't change the function of UI should be still valid?